src/main/resources/static/templates/UnionBasedSQLInjectionVulnerability/LEVEL_1/SQLInjection_Level1.js

```diff
@@ -1,18 +1,19 @@
-function addingEventListenerToFetchCarInfoButton() {
-  document
-    .getElementById("fetchCarImageButton")
-    .addEventListener("click", function () {
-      let url = getUrlForVulnerabilityLevel();
-      doGetAjaxCall(
-        fetchCarInfoCallBack,
-        url + "?id=" + document.getElementById("carId").value,
-        true
-      );
-    });
-}
-addingEventListenerToFetchCarInfoButton();
-
 function fetchCarInfoCallBack(data) {
-  document.getElementById("carInformation").innerHTML =
-    "<img src='" + data.imagePath + "' width='900'/>";
```

Comment:
CWE-79: Improper Neutralization of Input During Web Page Generation ('Cross-site Scripting')
------
User controlled data in methods like `innerHTML`, `outerHTML` or `document.write` is an anti-pattern that can lead to XSS vulnerabilities